src/videojs-contrib-hls.js

```diff
@@ -399,12 +399,51 @@ class HlsHandler extends Component {
           this.masterPlaylistController_.selectPlaylist = selectPlaylist.bind(this);
         }
       },
+      throughput: {
+        get() {
+          return this.masterPlaylistController_.mainSegmentLoader_.throughput.rate;
+        },
+        set(throughput) {
+          this.masterPlaylistController_.mainSegmentLoader_.throughput.rate = throughput;
+          this.masterPlaylistController_.mainSegmentLoader_.throughput.count = 1;
```

Comment:
Why does a reset of throughput set the count to 1, but reseting the bandwidth below resets the throughput to a count of 0?